src/bionics.cpp

```diff
@@ -85,20 +203,26 @@ bionic_data const& bionic_info(std::string const &id)
 
     debugmsg("bad bionic id");
 
-    static bionic_data const null_value {"bad bionic", false, false, 0, 0, 0, 0, 0, "bad_bionic", false};
+    static bionic_data const null_value {"bad bionic", false, false, 0, 0, 0, 0, 0,
+                                         "bad_bionic", false,
+                                         { num_bp , 1 } };
```

Comment:
Unfortunately, nothing helps me to deal with these errors and I got it again and again. Stackoverflow points out that it could be issues with libraries I used. I'll revert this part of PR for now since I don't want to be blocked due to this and will come back to it later.